<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>janelaPrincipal</class>
 <widget class="QMainWindow" name="janelaPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>607</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Usuario</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imagens.qrc">
    <normaloff>:/Imagens/user.png</normaloff>:/Imagens/user.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Usuario</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButtonCancelar">
         <property name="toolTip">
          <string>Cancelar</string>
         </property>
         <property name="toolTipDuration">
          <number>5000</number>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="imagens.qrc">
           <normaloff>:/Imagens/delete.png</normaloff>:/Imagens/delete.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="4">
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_Cpf_img">
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="imagens.qrc">:/Imagens/cpf.png</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_Cpf">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>15</height>
                </size>
               </property>
               <property name="text">
                <string>CPF</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_Cpf">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="inputMask">
              <string/>
             </property>
             <property name="maxLength">
              <number>11</number>
             </property>
             <property name="placeholderText">
              <string>CPF</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_Senha_img">
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="imagens.qrc">:/Imagens/password.png</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_Senha">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>15</height>
                </size>
               </property>
               <property name="text">
                <string>Senha</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_Senha">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>Senha</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_Email_img">
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="imagens.qrc">:/Imagens/email.png</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_Email">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>15</height>
                </size>
               </property>
               <property name="text">
                <string>Email</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_Email">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="placeholderText">
              <string>Email</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_Telefone_img">
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="imagens.qrc">:/Imagens/fone.png</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_Telefone">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>15</height>
                </size>
               </property>
               <property name="text">
                <string>Telefone</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_Telefone">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="inputMask">
              <string/>
             </property>
             <property name="maxLength">
              <number>14</number>
             </property>
             <property name="placeholderText">
              <string>Telefone</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0" colspan="2">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_Nome_img">
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="imagens.qrc">:/Imagens/user.png</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_Nome">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>15</height>
                </size>
               </property>
               <property name="text">
                <string>Nome</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_Nome">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="placeholderText">
              <string>Nome</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_PersistirUsuario">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Salvar</string>
         </property>
         <property name="toolTipDuration">
          <number>5000</number>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="imagens.qrc">
           <normaloff>:/Imagens/salvar.png</normaloff>:/Imagens/salvar.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>USUARIOS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QTableWidget" name="tableWidget_Listagem">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Nome</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>CPF</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>E-mail</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Telefone</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>492</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_Nome</tabstop>
  <tabstop>lineEdit_Cpf</tabstop>
  <tabstop>lineEdit_Senha</tabstop>
  <tabstop>lineEdit_Email</tabstop>
  <tabstop>lineEdit_Telefone</tabstop>
  <tabstop>pushButton_PersistirUsuario</tabstop>
  <tabstop>pushButtonCancelar</tabstop>
  <tabstop>tableWidget_Listagem</tabstop>
 </tabstops>
 <resources>
  <include location="imagens.qrc"/>
 </resources>
 <connections/>
</ui>
